plugins/light/service.js

```diff
@@ -187,7 +186,7 @@
 				update['bri'] = Math.round(setting['colorHSV'][2] * setting['brightness']);
 			}
 
-			$http.put('http://' + config.light.settings.hueIp + '/api/' + config.light.settings.hueUsername + "/groups/" + config.light.setup[index].targets[i].id + "/action", update)
+			$http.put('http://' + config.light.settings.hueIp + '/api/' + config.light.settings.hueUsername + "/lights/" + config.light.setup[index].targets[i].id + "/state", update)
```

Comment:
You're sure you want to drop the support for the groups of lights and speak to the lights directly? This might break some people's mirror configuration.

Reply:
Yikes, you are absolutely right! I'll revert this so we don't break folks. We should support both single lights and groups of lights.
Honestly this service is too messy/scary to edit - it'll probably have to be rewritten.